index.js

```diff
@@ -27,6 +28,9 @@ module.exports = (input, opts) => {
 	const runTests = !opts.yolo;
 	const runCleanup = !opts.skipCleanup && !opts.yolo;
 	const pkg = readPkgUp.sync().pkg;
+	if (!pkg) {
+		throw new Error(`${logSymbols.error} No package.json found. Make sure you're in the correct project.`);
+	}
```

Comment:
I think it would be cleaner if we extracted this in something like `lib/util.js`

```js
const pkg = util.readPkg();
```

@sindresorhus Any reason why we didn't added the error symbol yet in [cli.js](https://github.com/sindresorhus/np/blob/master/cli.js#L55)?